src/test/java/com/frameworkium/integration/ai/capture/api/tests/executions/Equality.java

```diff
@@ -0,0 +1,25 @@
+package com.frameworkium.integration.ai.capture.api.tests.executions;
+
+
+import com.frameworkium.integration.ai.capture.api.dto.executions.Execution;
+
+import static com.google.common.truth.Truth.assertThat;
+
+public class Equality {
```

Comment:
Would `assertThat(responseEx).isEqualTo(expectedEx)` not do the same thing as this? If not, could we make it such that it does? As I think this extra code may confuse people new.